src/components/Searchbar.js

```diff
@@ -1,15 +1,161 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
 import ArrowDropUpIcon from '@material-ui/icons/ArrowDropUp';
 import constants from '../styling/Constants';
 import { TextInput, DateInput, StyledButton } from '../components';
+import { useHistory } from 'react-router-dom';
+
+function getInputVal(wrapId) {
+    return document.getElementById(wrapId)?.getElementsByTagName('input')[0]?.value;
```

Comment:
let's put a function signature for each of these functions (comment on what the function does/why we need it)